scripts/setup_ubuntu.sh

```diff
@@ -2,6 +2,13 @@
 
 set -xeuo pipefail
 
+# Check the options
+INSTALL_CUDA=1
+if [ "${1-}" == "--no-cuda" ]; then
+  INSTALL_CUDA=
+  shift 1
+fi
```

Comment:
Can you have a `--cuda` flag instead of `--no-cuda`? Defaulting to adding cuda seems like a bad idea. Remember some people use this script to setup dev systems.

Reply:
Changed to `--with-cuda` flag.